j5/backends/hardware/sr/v4/power_board.py

```diff
@@ -234,56 +233,52 @@ def get_power_output_current(self, board: Board, identifier: int) -> float:
         current, = struct.unpack("<I", self._read(cmd))
         return cast(int, current) / 1000  # convert milliamps to amps
 
-    def buzz(self, board: Board, identifier: int,
-             duration: timedelta, pitch: Pitch) -> None:
+    def buzz(self, identifier: int,
+             duration: timedelta, pitch: int) -> None:
```

Comment:
If this function now only takes a frequency and no longer accepts a `Note`, please can rename this argument from `pitch` to `frequency`?

Reply:
This change was made in a separate PR, but I'll fix it here anyway